vms/volunteer/models.py

```diff
@@ -96,3 +96,6 @@ class Volunteer(models.Model):
         blank=True)
 
     user = models.OneToOneField(User)
+
+    def __unicode__(self):
+        return "%s %s" % (self.first_name, self.last_name)

```

Comment:
For next time, and even this time - its always better to use string formatting (would save time even when we upgrade at some point).

```
return '{0} {1}'.format(self.first_name, ... )
``` 
also, see that `' '` are used

Reply:
Ok @tonythomas01 . I am updating the PR.